registry/import.go

```diff
@@ -0,0 +1,153 @@
+package registry
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"encoding/json"
+	"fmt"
+	"github.com/cozy/cozy-apps-registry/asset"
+	"github.com/cozy/cozy-apps-registry/config"
+	"github.com/go-kivik/kivik"
+	"io"
+	"io/ioutil"
+	"path"
+	"strings"
+)
+
+func cleanCouch() error {
+	fmt.Printf("Clean CouchDB\n")
+	for _, db := range couchDatabases() {
+		name := db.Name()
+		fmt.Printf("  Clean CouchDB %s\n", name)
+		if err := client.DestroyDB(ctx, name); err != nil {
+			return err
+		}
+		if err := client.CreateDB(ctx, name); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func importCouchDocument(reader io.Reader, db string, id string) (string, error) {
+	fmt.Printf("Import CouchDB document %s.%s\n", db, id)
```

Comment:
Do we really need a message for each doc?

Reply:
Showing something is in progress + debug in case of failure ? :thinking: